common-ui-view/src/main/java/app/tivi/ui/SnackbarManager.kt

```diff
@@ -33,6 +36,23 @@ class SnackbarManager @Inject constructor() {
     private val pendingErrors = MutableSharedFlow<UiError>(extraBufferCapacity = 3)
```

Comment:
Does this work as you expect? Even if there are no collectors, messages will be sent and they could be potentially lost. This `extraBufferCapacity` is only helpful when `SharedFlow` has slow subscribers. `emit()` will suspend until all collectors receive the event.

Reply:
The following code buffers UiErrors in the channel when there are no collectors

```
private val _pendingErrors = Channel<UiError>(3, DROP_OLDEST)
val pendingErrors = _navigationEvents.receiveAsFlow().shareIn(externalScope, WhileSubscribed())
```

Reply:
Thanks, went back to using a Channel. Wasn't sure that we need the `shareIn()`. PTAL

Reply:
Looks good 👍  thank you!